File: capstone-backend-master/CustomerMs/src/main/java/in/team2/boot/CustomerMs/model/Customer.java
package in.team2.boot.CustomerMs.model;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.*;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.time.LocalDate;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Customer {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int customerId;
    private String name;
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
    private LocalDate dob;
    private String phoneNo;
    private String email;
    private String password;
    private String address;
    private int rewardPoints;
    private float walletAmount;
    private int walletLimit;
    private LocalDate createdAt;
}

File: capstone-frontend-master/src/app/customer.ts
export class Customer {
    customerId : number;
    name : string;
    dob : Date;
    phoneNo : string;
    email : string;
    password : string;
    address : string;
    rewardPoints : number;
    walletAmount : any;
    walletLimit : number;


    constructor(){
        this.customerId = 0;
        this.name = "Test";
        this.dob = new Date();
        this.phoneNo = "8800091211";
        this.email = "test@test.com";
        this.password = "Test@123";
        this.address = "Test, Test City, Test Country";
        this.rewardPoints = 100;
        this.walletAmount = 5000.99;
        this.walletLimit = 100000;
    }
}